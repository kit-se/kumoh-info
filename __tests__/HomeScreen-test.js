import 'react-native';
import React from 'react';
import Home from '../screens/HomeScreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <Home />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
