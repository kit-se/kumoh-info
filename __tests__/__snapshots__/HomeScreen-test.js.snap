// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "paddingTop": 80,
      }
    }
    style={
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "marginBottom": 20,
            "marginTop": 10,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 38,
              "marginLeft": -10,
              "marginTop": 3,
              "resizeMode": "contain",
              "width": 140,
            }
          }
        />
      </View>
    </View>
  </RCTScrollView>
  <View
    activeOpacity={1}
    style={
      Array [
        Object {
          "flex": 1,
          "flexDirection": "row",
        },
        undefined,
      ]
    }
  >
    <View
      activeOpacity={1}
      style={
        Array [
          Object {
            "flex": 1,
            "flexDirection": "column",
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
      >
        ddddd
      </Text>
    </View>
  </View>
</View>
`;
